src/ReleaseHistory.md

```diff
@@ -257,4 +257,8 @@
 * API BREAKING: define 'toolComponent' object to persist tool data. The 'tool.driver' component documents the standard driver metadata. 'tool.extensions' is an array of 'toolComponent' instances that describe extensions to the core analyzer. This change also deletes 'tool.sarifLoggerVersion' (from the newly created 'toolComponent' object) due to its lack of utility. Adds 'result.extensionIndex' to allow results to be associated with a plug-in. 'toolComponent' also added as a new file role. https://github.com/oasis-tcs/sarif-spec/issues/179
 * API BREAKING: Remove 'run.resources' object. Rename 'rule' object to 'reportingDescriptor'. Move rule and notification reportingDescriptor objects to 'tool.notificationDescriptors' and 'tool.ruleDescriptors'. 'resources.messageStrings' now located at 'toolComponent.globalMessageStrings'. 'rule.configuration' property now named 'reportingDescriptor.defaultConfiguration'. 'reportingConfiguration.defaultLevel' and 'reportingConfiguration.defaultRank' simplified to 'reportingConfiguration.level' and 'reportingConfiguration.rank'. Actual runtime reportingConfiguration persisted to new array of reportingConfiguration objects at 'invocation.reportingConfiguration'. https://github.com/oasis-tcs/sarif-spec/issues/311
 * API BREAKING: 'run.richTextMessageMimeType' renamed to 'run.markdownMessageMimeType'. 'message.richText' renamed to 'message.markdown'. 'message.richMessageId' deleted. Create 'multiformatMessageString' object, that holds plain text and markdown message format strings. 'reportingDescriptor.messageStrings' is now a dictionary of these objects, keyed by message id. 'reporting.Descriptor.richMessageStrings' dictionary is deleted. https://github.com/oasis-tcs/sarif-spec/issues/319
-* API BREAKING: 'threadflowLocation.kind' is now 'threadflowLocation.kinds', an array of strings that categorize the thread flow location. https://github.com/oasis-tcs/sarif-spec/issues/202
\ No newline at end of file
+* API BREAKING: 'threadflowLocation.kind' is now 'threadflowLocation.kinds', an array of strings that categorize the thread flow location. https://github.com/oasis-tcs/sarif-spec/issues/202
+
+## **v2.0.0-csd.2.beta.2019.01.24.1** [Sdk](https://www.nuget.org/packages/Sarif.Sdk/2.0.0-csd.2.beta.2019.01.24.1) | [Driver](https://www.nuget.org/packages/Sarif.Driver/2.0.0-csd.2.beta.2019.01.24.1) | [Converters](https://www.nuget.org/packages/Sarif.Converters/2.0.0-csd.2.beta.2019.01.24.1)) | [Multitool](https://www.nuget.org/packages/Sarif.Multitool/2.0.0-csd.2.beta.2019.01.24.1))
+* BUGFIX: 'region.charOffset' default value should be -1 (invalid value) rather than 0. Fixes an issue where 'region.charLength' is > 0 but 'region.charOffset' is absent (because its value of 0 was incorrectly elided due to being the default value). 
+* API BREAKING: rename 'result.kind' enum value 'none' to 'debug'
```

Comment:
>debug [](start = 59, length = 5)

I emailed to argue that we should not make this change. Please see email.

Reply:
yes, will proceed

---
In reply to: [259148424](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259148424) [](ancestors = 259148424)

Reply:
I will pull your branch and remove this.

---
In reply to: [259159295](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259159295) [](ancestors = 259159295,259148424)